packages/forms/resources/js/components/file-upload.js

```diff
@@ -317,6 +306,9 @@ export default function fileUploadFormComponent({
                     return
                 }
 
+                // Get the current order as displayed in FilePond and pass it to the backend.
+                await this.updateFileOrder()
```

Comment:
Can we skip this if there is only 1 file? Otherwise there is another network request for no reason.

Are there any other situations where it can be skipped?